mentat/config.py

```diff
@@ -83,6 +83,13 @@ class Config:
         },
         converter=converters.optional(converters.to_bool),
     )
+    auto_tokens: int = attr.field(
+        default=4000,
+        metadata={
+            "description": "The number of tokens auto-context can add.",
```

Comment:
Maybe rephrase this to "will add" since that's the current behavior roughly

Reply:
Only if it's turned on though, that was the 'can'

Reply:
On second thought I agree, will is clearer.